controllers/gateway_controller_test.go

```diff
@@ -10,15 +10,14 @@ import (
 	"github.com/rancher/opni/apis/v1beta2"
 	cfgv1beta1 "github.com/rancher/opni/pkg/config/v1beta1"
 	"github.com/rancher/opni/pkg/noauth"
-	"github.com/rancher/opni/pkg/test"
 	"github.com/rancher/opni/pkg/util"
 	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
 	rbacv1 "k8s.io/api/rbac/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-var _ = Describe("Gateway Controller", Ordered, Label(test.Controller, test.Slow), func() {
+var _ = Describe("Gateway Controller", Ordered, Label("controller", "slow"), func() {
```

Comment:
How come we're moving away from constants?

Reply:
The `ginkgo labels` command is used to filter tests by label, but it can only identify string constants since it works by parsing the AST. Running ginkgo with `--label-filter` (like we have been doing until now) won't work because it has to compile the tests to check the filter, and will delete the test binaries once it determines they should be skipped. This means we can't run two disjoint sets of tests at the same time, because one will end up deleting a binary the other set needs before it can run.